File: Lesson 1 - CPP and DSA Intro/GetFactorial/get_factorial.c
#include <stdio.h>

int main() {
    int n, factorial = 1, i = 1;

    printf("Enter value for n: ");
    scanf("%d", &n);

    while (i <= n) {
        factorial *= i;
        i++;
    }

    printf("Factorial: %d\n", factorial);
    return 0;
}

File: Lesson 1 - CPP and DSA Intro/GetFactorial/getFactorial.js
const readline = require("readline");

const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout,
});

rl.question("Enter value for n: ", (input) => {
    let n = parseInt(input);
    let factorial = 1;
    let i = 1;

    while (i <= n) {
        factorial *= i;
        i++;
    }

    console.log("Factorial: " + factorial);
    rl.close();
});

/*
    To run this code, make sure that you are in its directory. 
    After that, use the command: node getFactorial.js

*/